lib/bottombar.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
```

Comment:
Don't need this here. main() is only used for the main file of the app (in our case main.dart)